parser/types.go

```diff
@@ -80,10 +79,11 @@ type BurnFee struct {
 }
 
 type FeesMetadata struct {
-	TxType                string
+	TxType                string `json:"TxType,omitempty"`
```

Comment:
the json should be tx_type, as that is how it is inserted on the db

Reply:
<img width="454" alt="Screenshot 2025-03-21 at 19 27 11" src="https://github.com/user-attachments/assets/3b40d0c9-bc20-419c-8c1f-9456dc24ace6" />
right now its been saved as TxType